superset/views/core.py

```diff
@@ -285,6 +294,7 @@ class DatabaseView(SupersetModelView, DeleteMixin, YamlExportMixin):  # noqa
         'allow_multi_schema_metadata_fetch': _('Allow Multi Schema Metadata Fetch'),
         'backend': _('Backend'),
     }
+    base_filters = [['id', DatabaseFilter, lambda: []]]
```

Comment:
I think this broke stuff on our end, users can see DBs in SQL Lab. I commented it out and permissions work correctly.

Reply:
Do you mean "can't see" (you wrote "can see").  This may require adding `all_database_access` to `alpha`

Reply:
https://github.com/apache/incubator-superset/pull/7271